.github/workflows/master.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         go-version: [ 1.15.x ]
-        python-version: [ 3.7.x ]
+        python-version: [ 3.9.x ]
```

Comment:
Not really a comment for this PR, but hopefully as part of https://github.com/pulumi/pulumi/issues/5664 we can find some way to have coverage of all the versions of Python we support (e.g. 3.6.x, 3.7.x, 3.8.x, and 3.9.x). Maybe as some cron run, and keep the matrix smaller for PRs? cc: @stack72

Reply:
Yeah I agree, I had initially run this as a matrix using `[3.7.x, 3.9.x]` - I think it makes sense to do that as a cron.